### PR TITLE
get rid of commitlint

### PR DESCRIPTION
**Pull request by ofrobots**
We tried this experiment, I am not sure it is successful:
Too onerous. Unfriendly to non-collaborators. Doesn't work well with github squash and rebase workflow.

Opinions @GoogleCloudPlatform/node-team ?

### CODE REVIEW

**Comment by kjin** [comment]
LGTM. We should still follow conventional commits, especially since we've already got the ball rolling... mostly. It should be possible today to write a GitHub integration to enforce this by failing a check if the PR's title doesn't satisfy conventional commits, since squash+merge'd commits take on the title of the PR itself. Enforcing the body is a little different issue... I've emailed GitHub support about whether there's a feature for Squash and Merge to take on the contents of the initial PR description.

So in short, I think we should still enforce this, but at the GitHub integrations level rather than at the git CLI level.

File: package.json [hunk 1]
```diff
@@ -10,8 +10,7 @@
     "changelog": "./bin/run-changelog.sh",
     "coverage": "./bin/run-test.sh -c",
     "bump": "./bin/run-bump.sh",
-    "closure": "./node_modules/.bin/closure-npc",
-    "commitmsg": "commitlint -e"
+    "closure": "./node_modules/.bin/closure-npc"
```
Comment (matthewloring) [inline package.json]: Just since it's highlighted in this change, should we remove this at some point?

Comment (ofrobots) [reply package.json]: Done.

**Comment by AdriVanHoudt** [comment]
There is https://github.com/conventional-changelog/validate-commit-msg just an fyi

**Comment by ofrobots** [comment]
@AdriVanHoudt Thanks! The main challenge we saw was that commit hook time validation puts the burden on people trying to contribute to the repo. In reality the commit time message doesn't matter but rather the message we use at squash-and-merge time in GitHub, which does not get validated.

**Comment by marionebl** [comment]
Sorry to read `commitlint` did not match your use case. Paul Irish recently started work on https://github.com/paulirish/commitlintbot. Have not tried it myself but it looks as if this could solve the Squash/Merge problem.

**Comment by kjin** [comment]
Thanks for pointing us in that direction! Looking at the high level
description it seems like this could be very useful to us.
